emeis/settings.py

```diff
@@ -145,6 +146,7 @@ def parse_languages(languages):
 LANGUAGES = (
     parse_languages(env.list("LANGUAGES", default=["en"])) or global_settings.LANGUAGES
 )
+LOCALE_PATHS = [django_root("emeis", "locale")]
```

Comment:
This should be env-configurable as well, for when Emeis is embedded in other apps as python package